apps/web/src/views/universalFarms/components/PoolsFilterPanel.tsx

```diff
@@ -145,10 +150,17 @@ export const PoolsFilterPanel: React.FC<React.PropsWithChildren<IPoolsFilterPane
     onChange({ selectedTokens: e.value })
   }
 
+  const childrenCount = useMemo(() => 3 + React.Children.count(children), [children])
```

Comment:
hmm, I think it is not safe, think this code as children. why not just pass a `count` props?

```tsx

<PoolsFilterPanel>
  <>
    <Children1 />
    <Children2 />
    <Children3 />
  </>
</PoolsFilterPanel>

```

react.children only 1, but it rendered 3 items

Reply:
I think we can provide a default resolution to cover the common cases and reduce the usage to care about it. 
in this case, count props not work either because the Children1,2,3 not the PoolsFilterPanel's children